protos/memcache/text.go

```diff
@@ -0,0 +1,744 @@
+package memcache
+
```

Comment:
Having a brief description/documentation what this file is about would be nice. It was not obvious to me at first.